test/test_snapshot_manager.py

```diff
@@ -0,0 +1,38 @@
+import ebs_snapshots.snapshot_manager as snapshot_manager
+import unittest
+import boto
+from moto import mock_ec2
+
+class TestSnapshotManager(unittest.TestCase):
+
+    def _mock_true(*args, **kwargs):
+        return True
```

Comment:
could use unittest mocks, e.g. if you want to assert that these methods get called: https://docs.python.org/3/library/unittest.mock.html#quick-guide